PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+PULL REQUEST TEMPLATE
+======================
+
```

Comment:
Could you add the line:

Fixes # (issue number, if applicable)

Reply:
done